src/GitHub_Updater/Base.php

```diff
@@ -1270,6 +1270,9 @@ public function get_running_git_servers() {
 			if ( ! empty( $e->enterprise ) && false !== stripos( $e->type, 'bitbucket' ) ) {
 				return 'bbserver';
 			}
+			if (! empty( $e->enterprise ) && 'gitlab.com' !== $e->enterprise ) {
+				return 'gitlab-selfhosted';
+			}
```

Comment:
Let’s stick to the same logic for the conditional using `stripos( $e->type, ‘gitlab’ )` and return `gitlab_enterprise` as above.